src/sentry/static/sentry/app/components/buttons/button.jsx

```diff
@@ -44,11 +44,17 @@ class Button extends React.Component {
      */
     label: PropTypes.string,
 
+    /**
+     * flexbox justify property to use to align label
+     */
+    alignLabel: PropTypes.string,
+
     onClick: PropTypes.func,
   };
 
   static defaultProps = {
     disabled: false,
+    alignLabel: 'flex-start',
```

Comment:
Can we pass a `node` instead of a string to `label` and just style that? It will make controlling the look of this component way easier. otherwise we are gonna keep adding stuff like this I can see it 🔮🔮🔮🔮

Reply:
or if we don't do that, lets at least change it to ['left', 'center', 'right']